CONTRIBUTING.md

```diff
@@ -60,25 +62,25 @@ This section describes how to build and run the Botkube from the source code.
      docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
      docker push <your_account>/botkube:v9.99.9-dev
      ```
-     Where `<your_account>` is Docker hub account to which you can push the image.
+     Where `<your_account>` is Docker hub or any other registry provider account to which you can push the image.
+
+2. Ensure you have a Slack installation already setup:
```

Comment:
I'd merge point 2 and 3 and have a single item and make it even more general - after all the Slack integration is not required for the development setup. What do you think about something like:

```suggestion
2. Install Botkube with any of communication platform configured, according to [the installation instructions](https://botkube.io/docs/installation/). During the Helm chart installation step, set the following flags:
```

+ removing sections below (suggestions incoming as well)